Xamarin.Essentials/Compass/Compass.android.cs

```diff
@@ -52,8 +53,43 @@ internal static void PlatformStop()
         }
     }
 
+    class LowPassFilter
```

Comment:
I think that since this is generic it could be moved over to the .shared.cs file perhaps.